code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -842,7 +842,7 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/toxin/coniine/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
-	M.losebreath += 5 * REM * delta_time
+	M.losebreath = min(M.losebreath + 5 * REM * delta_time, 5)
```

Comment:
we need to make sure we aren't indirectly healing people with more than 5 losebreath prior to having this drug in their system.

Reply:
what about
```suggestion
        var/amount= 5 * REM * delta_time
        if(M.losebreath < amount)
            M.losebreath += amount
```